mshv-bindings/Cargo.toml

```diff
@@ -6,11 +6,12 @@ edition = "2018"
 license = "Apache-2.0 OR BSD-3-Clause"
 
 [features]
-fam-wrappers = ["vmm-sys-util"]
+fam-wrappers = []
 with-serde = ["serde", "serde_derive"]
 
 [dependencies]
+libc = ">=0.2.39"
 serde = { version = ">=1.0.27", optional = true }
 serde_derive = { version = ">=1.0.27", optional = true }
-vmm-sys-util = { version = ">=0.2.0", optional = true }
+vmm-sys-util = ">=0.2.1"
```

Comment:
I can see why `libc` would be needed here, but why the other two changes needed?

Reply:
Now we need vmm-sys-util for Result type. That is not optional. So we need vmm-sus-util as required dependency and hence make independent of fam-wrappers feature.